exchangelib/services.py

```diff
@@ -872,3 +877,81 @@ def _get_payload(self, items):
             raise AttributeError('"ids" must not be empty')
         payload.append(attachment_ids)
         return payload
+
+
+class ExportItems(EWSPooledAccountService, ExpectResponseErrorsMixin):
+    """
+    MSDN: https://msdn.microsoft.com/en-us/library/office/ff709523(v=exchg.150).aspx
+    """
+    CHUNKSIZE = 100
+    SERVICE_NAME = 'ExportItems'
+    element_container_name = "{%s}Data" % MNS
+
+    def call(self, item_ids):
+        return self._pool_requests(
+            payload_func=self._get_payload, items=item_ids, version=self.account.version
+        )
+
+    def _get_payload(self, items, version):
+        from .folders import ItemId
+        exportitems = create_element('m:%s' % self.SERVICE_NAME)
+        itemids = create_element('m:ItemIds')
+        exportitems.append(itemids)
+        for item_id in items:
+            if isinstance(item_id, ItemId):
```

Comment:
This duplicates code unnecessarily. Existing services do this:

```
for item in items:
    item_id = ItemId(*(item if isinstance(item, tuple) else (item.item_id, item.changekey)))
    itemids.append(itemid)
```

which supports both `Item`, `ItemId` and `tuple` input.